synapse/lib/spawn.py

```diff
@@ -447,21 +454,58 @@ async def bumpSpawnPool(self):
     async def getStormPkgs(self):
         return list(self.stormpkgs.values())
 
+    async def _addStormSvc(self, sdef):
+
+        iden = sdef.get('iden')
+        ssvc = self.svcsbyiden.get(iden)
+        if ssvc is not None:
+            return ssvc.sdef
+
+        ssvc = await self._setStormSvc(sdef)
+
+        return ssvc.sdef
+
+    async def _delStormSvcPkgs(self, iden):
+        pass
```

Comment:
Should this raise something like "package deletion not supported from spawn cortexes?"

Reply:
That's actually not for package deletion. That's the part that removes service packages before installing the service again. In the spawn, we don't want loading a service to impact the main hive, which is what this accomplishes.